onnxruntime/core/providers/cpu/math/matmul.h

```diff
@@ -15,6 +15,14 @@ class MatMul final : public OpKernel {
   Status Compute(OpKernelContext* context) const override;
 };
 
+static inline bool ort_use_fastmath_mode() {
```

Comment:
For compile time feature switch, use a compilation flag. For runtime feature switch, use a session option.

https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

Reply:
Thanks, @chenfucn , I will take a loot at the session options.